.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "pre-commit": "vue-cli-service lint --no-fix"
```

Comment:
I think that this is an ok place to do the fix? As long as it is happening on the dev's machine I think it's ok to auto-fix.

Reply:
Yeah -- I tested it out locally and I think this is what we want, changing this to `--fix`.